Jenkinsfile

```diff
@@ -26,6 +26,7 @@ pipeline
 						sh "bash ./build.sh ${env.BUILD_NUMBER} deb"
 						deployDeb dir: "build-results_deb", map_repo: pkg_map_branches('focal-agent'), user: "rbrepo", agent: "rep-agent"
 						deployDeb dir: "build-results_deb", map_repo: pkg_map_branches('jammy-agent'), user: "rbrepo", agent: "rep-agent"
+						deployDeb dir: "build-results_deb", map_repo: pkg_map_branches('bullseye-agent'), user: "rbrepo", agent: "rep-agent"
```

Comment:
@kgermanov Is it ok, that we build a package for Debian in a container based on the Ubuntu image?